### DISCUSSION TITLE
How to specify a global access rule to the batch compiler?

### DISCUSSION THREAD

**Discussion by laeubi** [eclipse-jdt/eclipse.jdt.core#1945] [Q&A]
https://help.eclipse.org/latest/topic/org.eclipse.jdt.doc.user/tasks/task-using_batch_compiler.htm explains how one can specify access rules for files and folder, but for example here we have a global rule:

https://github.com/eclipse-pde/eclipse.pde/blob/cef50ec845862cdb1b1f3ea5f347380bc0e467b6/e4tools/bundles/org.eclipse.e4.tools.emf.ui/.classpath#L6

the question is now how one would pass this to the batch compiler? Must I prepend it to each and every jar/file to be effective or can there be a wildcard used or ...?

**Comment by guw** [comment]
The Batch compiler does not know about the concept of containers which group jars. Thus, it needs to be applied to every jar.
